R/Driver.R

```diff
@@ -37,48 +37,63 @@ setMethod(
+#' @param server The server hostname. Some drivers use `Servername` as the name
+#' for this argument. Not required when configured for the supplied `dsn`.
+#' @param database The database on the server. Not required when configured for
+#' the supplied `dsn`.
+#' @param uid The user identifier. Some drivers use `username` as the name
+#' for this argument. Not required when configured for the supplied `dsn`.
+#' @param pwd The password. Some drivers use `password` as the name
+#' for this argument. Not required when configured for the supplied `dsn`.
 #' @param dbms.name The database management system name. This should normally
 #' be queried automatically by the ODBC driver. This name is used as the class
-#' name for the OdbcConnect object returned from `dbConnect()`. However if the
-#' driver does not return a valid value it can be set manually with this
+#' name for the OdbcConnection object returned from [dbConnect()]. However, if
+#' the driver does not return a valid value, it can be set manually with this
 #' parameter.
 #' @param attributes An S4 object of connection attributes that are passed
-#' prior to the connection being established.  See \link{ConnectionAttributes}.
-#' @param ... Additional ODBC keywords, these will be joined with the other
+#' prior to the connection being established. See \link{ConnectionAttributes}.
+#' @param ... Additional ODBC keywords. These will be joined with the other
 #' arguments to form the final connection string.
 #' @param .connection_string A complete connection string, useful if you are
-#' copy pasting it from another source. If this argument is used any additional
+#' copy pasting it from another source. If this argument is used, any additional
 #' arguments will be appended to this string.
-#' @param bigint The R type that `SQL_BIGINT` types should be mapped to,
-#' default is [bit64::integer64], which allows the full range of 64 bit
+#' @param bigint The R type that `SQL_BIGINT` types should be mapped to.
+#' Default is [bit64::integer64], which allows the full range of 64 bit
 #' integers.
 #' @param timeout Time in seconds to timeout the connection attempt. Setting a
-#' timeout of `Inf` indicates no timeout. (defaults to 10 seconds).
+#' timeout of `Inf` indicates no timeout. Defaults to 10 seconds.
 #' @details
 #' The connection string keywords are driver dependent. The parameters
 #' documented here are common, but some drivers may not accept them. Please see
-#' the specific driver documentation for allowed parameters,
+#' the specific driver documentation for allowed parameters;
 #' \url{https://www.connectionstrings.com} is also a useful resource of example
 #' connection strings for a variety of databases.
+#'
+#' Note that all generic ODBC parameter names, and the values of `drv` and
+#' `dsn`, are case-insensitive. The case-sensitivity of the values of generic
+#' arguments, as well as the names and values of driver-specific arguments,
+#' are determined by the driver `drv`.
```

Comment:
https://learn.microsoft.com/en-us/openspecs/sql_server_protocols/ms-odbcstr/0eb8c7e5-93a8-4d7c-8ae6-abdf37df5df3

Reply:
See also #641.

But it might be better to put this in the docs for `...`?

Reply:
Ah, good point!

Reply:
[5afcce0](https://github.com/r-dbi/odbc/pull/647/commits/5afcce05f343840718fa13ef880b53741e295070).